enhancements/ota/distribute-secondary-metadata-as-container-image.md

```diff
@@ -0,0 +1,165 @@
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [x] Graduation criteria
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Open Questions
+
+### Cincinnati Git(Hub) scraper deprecation
+We need to decide whether we want to maintain the Git(Hub) scraper plugin in the future or drop it altogether.
+
+
+### Content trust
+Trust in the container image registry and in the container image content is possibly an issue if we an approach where Cincinnati uses a mutable tag, e.g. `latest` tag to receive the most recent content from the container registry.
+Assuming we're using Quay as an image registry, [which does not support Docker Content Trust](https://support.coreos.com/hc/en-us/articles/115000289114-Docker-Content-Trust-support).
+If we decide that to ensure the image integrity, which means do we have which do not further complicate the workflow?
+
+### Product name for the CPaaS integration
+Which product name should this image be released under?
+Related to this is that it's currently unclear if the Cincinnati image itself will be published under its own product name or now.
+Further note: the repository for the graph-data images will not be semantically versioned.
+
+### Repository location and name
+Under which well-known container image repository will we publish the images?
+
+
+### Continuous build and release
+Can CPaaS handle continuous build and release from master?
+
+## Summary
+The [cincinnati-graph-data GitHub repository][cincinnati-graph-data] will be continuously packaged and released in form of a container image by the means of integration with CPaaS.
+This image will be available for public consumption, and will be used by the publicly available Cincinnati instances operated by Red Hat.
+The image can be transferred and served to Cincinnati instances operated by any party, including but not limited to network-restricted environments.
+The source of the container image, namely the cincinnati-graph-data, will remain publicly available and its workflows unaffected by this enhancement.
+
+## Motivation
+
+### Goals
+- The container image will become the primary way of consuming and distributing the content of the cincinnati-graph-data repository.
+- As long as the primary release metadata will be packaged, distributed and consumed in form of container images as well, this streamlines all transport mechanisms required for data consumption by Cincinnati, as well as mirroring and serving its consumed content.
+- Every change to the master branch of the cincinnati-graph-data results in a new container image containing the unmodified working tree files of the new revision.
+- The image will be pushed to a publicly available container image registry und a well-known name.
```

Comment:
"und" -> "with".  And this seems more like "proposal" than "goals".  Goals would be:

* Unify the transport mechanism for primary and secondary data.
* Allow for consumption by both internal and external consumers.
* Do not modify the existing graph-data maintainer workflows.

or some such.

Reply:
"und" was meant to be "under". Is "with" more appropriate linguistically?

Reply:
on "goals" vs "proposal", I've been unsure about the structure of the document. The template text made it seem to me that this is the section where the proposed changes are given. Now that you're saying it, I understand that's somewhat contradictory with the existence of a "proposal" section :smile:

Reply:
> Is "with" more appropriate linguistically?

Both "under" or "with" would work, I don't see much difference.